config/open-oracle.js

```diff
@@ -57,35 +88,17 @@ define("OpenOracle", {
 
     let zeroAddress = '0x0000000000000000000000000000000000000000';
 
-    async function getOrCreatePair(asset0, asset1, allowCreate=true) {
-      let pair = await read(uniswap, 'getPair(address,address): address', [asset0, asset1]);
-      if (pair === zeroAddress) {
-        if (allowCreate) {
-          console.log(`Creating Uniswap Pair: ${asset0} ${asset1}`);
-          await trx(uniswap, 'createPair(address,address)', [asset0, asset1]);
-          // TODO: We need to fund liquidity for the pair as well
-          return await getOrCreatePair(asset0, asset1, false);
-        } else {
-          throw new Error(`Could not find Uniswap pair ${asset0} ${asset1}`);
-        }
-      } else {
-        let reversed = bn(asset0).lt(bn(asset1)); // TODO: Check
-        return [ pair, reversed ];
-      }
-    }
-
     let configs = await config.reduce(async (accP, conf) => {
       let acc = await accP; // force ordering
       let uniswapMarket;
       let isUniswapReversed;
       if (conf.price_source === 'REPORTER') {
-        if (conf.symbol === 'ETH') {
-          [uniswapMarket, isUniswapReversed] = await getOrCreatePair(weth, usdc);
-        } else {
-          [uniswapMarket, isUniswapReversed] = await getOrCreatePair(conf.underlying, weth); // TODO: Check asset order
-        }
+        // TODO: Do we calculate isUniswapReversed?
```

Comment:
no, there is no guarantee this calculation will be the same as for mainnet pairs I am mocking.